### PR TITLE
Adding CHANGELOG

### PR DESCRIPTION
**Pull request by DLBPointon**
Updates the change log

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,14 +3,47 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.0dev - [date]
+## v1.0 - UNSC Infinity - [2023-10-02]
```
Comment (priyanka-surana) [inline CHANGELOG.md:6]:
```suggestion
## [[1.0.0](https://github.com/sanger-tol/curationpretext/releases/tag/1.0.0)] - UNSC Infinity - [2023-10-02]
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,14 +3,47 @@
...
 
-### `Fixed`
+- Subworkflow to generate tracks containing telomeric sites.
+- Subworkflow to generate Pretext maps and images
+- Subworkflow to generate repeat density tracks.
+- Subworkflow to generate longread coverage tracks from pacbio data.
+- Subworkflow to generate gap tracks.
 
-### `Dependencies`
+### Parameters
```
Comment (priyanka-surana) [inline CHANGELOG.md:19]: List all parameters in new version, similar to how you are doing for software dependencies.

**Review by priyanka-surana — CHANGES_REQUESTED** [review]
A couple minor changes, otherwise good to go.

File: CHANGELOG.md (hunk 2 above)
Comment (DLBPointon) [reply CHANGELOG.md:19]: Done

**Comment by DLBPointon** [comment]
Hi @priyanka-surana 
Fixed the above, this also includes the fixes for testing simply because I forgot to check my branch.

**Review by priyanka-surana — APPROVED** [review]
LGTM 👍🏼 Thank you for making the changes.

**Pull request merged**